interop/cats/src/main/scala/caliban/interop/cats/CatsInterop.scala

```diff
@@ -28,4 +32,20 @@ object CatsInterop {
     Async[F].async { cb =>
       runtime.unsafeRunAsync(graphQL.execute(query))(exit => cb(exit.toEither.void))
     }
+
+  def schema[F[_]: Effect, R, A](implicit ev: Schema[R, A]): Schema[R, F[A]] =
+    new Schema[R, F[A]] {
+      override def toType(isInput: Boolean): __Type =
+        ev.toType(isInput)
+
```

Comment:
Can you also override `optional` just like here: https://github.com/ghostdogpr/caliban/blob/master/core/src/main/scala/caliban/schema/Schema.scala#L227